contrib/mongodb/mongo-go-driver/mongo/mongo.go

```diff
@@ -0,0 +1,104 @@
+// Package mongo provides functions to trace the mongodb/mongo-go-driver package (https://github.com/mongodb/mongo-go-driver).
+//
+// `NewMonitor` will return an event.CommandMonitor which is used to trace
+// requests.
+package mongo
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/mongodb/mongo-go-driver/core/event"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// The values of these mongo fields will not be scrubbed out. This allows the
+// non-sensitive collection names to be captured.
+var unscrubbedFields = []string{
+	"ordered", "insert", "count", "find", "create",
+}
+
+type spanKey struct {
+	ConnectionID string
+	RequestID    int64
+}
+
+type monitor struct {
+	sync.Mutex
+	spans map[spanKey]ddtrace.Span
+}
+
+func (m *monitor) Started(evt *event.CommandStartedEvent) {
+	hostname, port := peerInfo(evt)
+	statement := scrub(evt.Command).ToExtJSON(false)
+
+	span, _ := tracer.StartSpanFromContext(evt.Context, "mongodb.query",
+		tracer.ServiceName("mongo"),
```

Comment:
We need to add `tracer.SpanType(ext.SpanTypeMongodb)` (add it to `ddtrace/ext/app_types.go`)

Reply:
I have this added in my PR.

Reply:
I just now noticed that this comment is a week old. :)